scripts/generate.sh

```diff
@@ -26,45 +26,45 @@ npx @openapitools/openapi-generator-cli version-manager set 5.2.0
 echo "Generating SDKs"
 additional_properties="generateInterfaces=true,enumClassPrefix=true"
 
-OPENAPI_FILENAME=".openapi/kas-fleet-manager.yaml"
-PACKAGE_NAME="kafkamgmtclient"
-OUTPUT_PATH="kafkamgmt/apiv1/client"
+# OPENAPI_FILENAME=".openapi/kas-fleet-manager.yaml"
```

Comment:
Temporary changes have been accidentally pushed, need to be removed for this PR.